### ISSUE TITLE
About Us Page 

### ISSUE THREAD

**Issue by mavish014** [moviepediafilms/moviepediafilms.com#28]
**Tagline:**
Let the films do the talking.

**Vision:** 
We aspire to democratize access to independent films and redefine filmmaking.

**Mission:** 
Our mission is to facilitate filmmaking processes and ensure that the best films move their audience. We strive to impact the lives of 10 million filmmakers.

Films have been the most effective medium of story-telling, impacting lives across the globe. A good story usually transcends boundaries. There’s a treasure trove of such films remaining obscure on the internet. Filmmakers, story-tellers burn their midnight oil for the reach while cinephiles get a headache browsing content. At Moviepedia Films, we strive to bridge this gap. We connect the creators with the enthusiasts. We ensure that a filmmaker’s time is spent more on the art while we seamlessly manage the daily hassles.
 
**What do we do?**
We curate films from across the globe. We use new-age technology to help our filmmakers reach out to a large pool of cinephiles to showcase their stories. We bridge the gap between enthusiasts and creators. Both can connect on any topic filmmaking. We ensure that every person gets to see refreshing films every day, which are otherwise hidden gems.
 
**Who are we?**
We are a bunch of film enthusiasts, binge-watching all the great content. We find a strong connection with the art of storytelling. We feel that a story can change the world. We derive our philosophy of life from films that move us. We are a community of filmmakers, actors, scriptwriters. We intend to spread the nuances of art across the world.
 
**Why us?**
There’s a bunch of middle-man standing between a filmmaker and the audience. We ensure that good stories reach out to their audience with the minimum hassle. We also do not ask filmmakers to give us their content rights. Instead, we make the process of filmmaking simpler.
 
**What’s there for you?**
You get hand-picked films recommended by industry veterans at Moviepedia Films, personalized just for you. We don’t charge you for the content. We connect you with your favorite filmmakers to contribute to their upcoming film projects. It doesn’t end there. You get to be the much coveted ‘Co-Producer’ of the film. From pre-production to post-production, you can stay involved in the process of filmmaking.

Are you a filmmaker? If the answer is yes, do check out how we can serve you:

Link: <Submit Film Page>

**Careers**
**Culture:** At Moviepedia Films, we draw inspiration from films. We firmly believe films are liberating, refreshing, and can impact billions of lives. To find such hidden gems, which are otherwise getting obscured on the internet, involves tireless efforts and a passion for creative arts. Another thing, it’s all about ‘US’ and never ‘me’ at Moviepedia Films.

**Values:** We are a people-first organization. Empathy forms the core. Be it a filmmaker or an employee or the audience, we ensure the best experience for our people. We are passionate and self-motivated, aiming to touch a billion lives. Our idea was born to bridge the transparency voids. Transparency is our top of the head priority. We emphasize mutual respect and empower each other. 

**Available roles:**

**Content Expert:** If you have the ability to binge-watch shows, then you’re at the right place. You’ve to sit back, watch movies, and share your critical views. You’d be responsible for analyzing the content and decide whether it fits the kind of content that we put up. 

Do you think you’re the one? 
If yes, send us your application at contactus@moviepediafilms.com

**Comment by durbar94** [comment]
Let the films do the talking.

We are the first social film discovery platform. We started as a group of filmmakers and cinephiles. Binge-watching and imagining screenplay went hand-in-hand. Soon, we realized that we are a mass of people who can move the world with the art of narration, and Moviepedia Films emerged.

Films have been the most effective medium of thought expression, impacting lives across the globe. A good story usually transcends boundaries. There’s a treasure trove of such films remaining obscure on the internet. We understand the pain a filmmaker faces in building his/her audience across an array of social media platforms. Film enthusiasts say that they spend most of their time browsing for films. It’s weird that despite having 500 hours of videos uploaded every minute on social media, it’s getting harder for enthusiasts to pick the best stories.

It is then we decided to build a platform for both the creator and curator. We strive to democratize access to quality content and lead the next wave of film-first community thinking.

Vision: We aspire to lead the next wave of film first community thinking.

Mission: Our mission is to democratize filmmaking and stream over 10 million films across the globe.


What do we do?

We are building a film-first community, where filmmakers get to screen their films directly. On the other hand, we democratize access to exciting plots. We involve the viewers to curate their film lists specifically for their friends and fellow film buffs. We believe both filmmaking and curation are art. We reward the filmmakers as well as the curators. We also provide a platform for discussions related to films and connect with top-notch industry veterans.

 

Who are we?

We are a bunch of film enthusiasts, binge-watching all the great content. We find a strong connection with the art of storytelling. We feel that a story can change the world. We derive our philosophy of life from films that move us. We are a community of filmmakers, actors, scriptwriters. We intend to spread the nuances of art across the world.

 

Why us?

There’s a bunch of middle-man standing between a filmmaker and the audience. We ensure that good stories reach out to their audience with the minimum hassle. We also do not ask filmmakers to give us their content rights. Instead, we make the process of filmmaking simpler.


 Are you a filmmaker? If the answer is yes, do check out how we can serve you:

Link: <Submit Film Page>
